src/main/resources/reference.conf

```diff
@@ -87,4 +87,10 @@ serviceAccounts {
     # This option is supported for ANY provider implementation. If not specified, the default is 30 seconds.
     providerTimeout = 30 seconds
   }
+}
+
+autoFreeze {
+  dateAccessedMonitorScheduler = 1 m
```

Comment:
Same comments as above, use full time unit names and change autoFreeze to 30 minutes